Content.Client/_NF/Latejoin/VesselListControl.xaml.cs

```diff
@@ -52,11 +52,32 @@ protected override void Dispose(bool disposing)
 
     private int DefaultComparison(NetEntity x, NetEntity y)
     {
+        // Ensure "Frontier Outpost" always appears on top
+        if (_gameTicker.StationNames[x] == "Frontier Outpost" && _gameTicker.StationNames[y] != "Frontier Outpost")
```

Comment:
It seems bad practice to me to have `"Frontier Outpost"` as string literals here – repeated four times, no less. If Frontier Outpost ever changes names, this code will stop working. At the very least, to reduce repetition and thereby risk of error, consider putting the name in a const somewhere and using that instead.

Reply:
You can get frontier by scanning for the only grid that has protected grid comp and jobs comp.
As it will omit all ships and POI

Please use it so its not hardcoded name and this PR should be good to go.